openmc/universe.py

```diff
@@ -118,6 +128,12 @@ def name(self, name):
         else:
             self._name = ''
 
+    @volume.setter
+    def volume(self, volume):
+        if volume is not None:
+            cv.check_type('universe volume', volume, Real)
```

Comment:
I don't think that it is standard convention to use the class name in the type check methods for the rest of our properties. Do you think that this is what we should move towards?